### PR TITLE
feat: put Skåne inside a region

### LINKED ISSUE #211: Put Helsingborg inside a region

### PR DESCRIPTION
**Pull request by alexanderczigler**
This changes how municipalities are instantiated. Rather than being a separate stream instantiated by the simulator, municipalities are now a part of a `Region`.

**Testing**
Testa köra i norrbotten resp. skåne. Du kan behöva ändra parameters.json för att ställa in vilka typer av strömmar som visas i respektive experiment. 

---
resolves #211

### CODE REVIEW

File: packages/simulator/streams/kommuner.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ function read({ fleets }) {
             ? merge(bookings.hm, bookings.ikea)
             : of(),
         })
-        kommun.citizens = getCitizens(kommun)
+        kommun.citizens = [] // TODO: Fix this :o)
```
Comment (alexanderczigler) [inline packages/simulator/streams/kommuner.js:118]: This is a temporary change, waiting for this to be fixed via another PR.

File: packages/simulator/streams/regions/skane.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  )
+
+  return new Region({
+    id: 'skane',
+    cats: 4,
+    name: 'Skåne',
+    kommuner: municipalities,
+
+    // NOTE: No buses in Skåne.
+    stopTimes: new Subject(),
```
Comment (irony) [inline packages/simulator/streams/regions/skane.js:18]: Ersätt med of() eller from([]) istället?

**Review by irony — APPROVED** [review]
Bravo!
